tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import random
+import datetime
+import allure
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.clock.ClockConsts import ClockConsts
+from tests.common.errors import RunAnsibleModuleFail
+
+
+class ClockUtils:
+    @staticmethod
+    def run_cmd(duthosts, cmd, param=''):
+        """
+        @summary:
+            Run a given command and return its output.
+            * A successful command returns an empty output (''), while failure returns an error message
+        @return: commands output (str)
+        """
+        with allure.step(f'Run command: "{cmd}" with param "{param}"'):
+            logging.info(f'Run command: "{cmd}" with param "{param}"')
+            DUT_HOSTNAME = duthosts[0].hostname
+
+            cmd_to_run = cmd if param == '' else cmd + ' ' + param
+            logging.info(f'Actual command to run: "{cmd_to_run}"')
+
+            try:
+                cmd_output = duthosts.command(cmd_to_run)[DUT_HOSTNAME][ClockConsts.STDOUT]
+            except RunAnsibleModuleFail as cmd_err:
+                output = cmd_err.results[ClockConsts.STDOUT]
+                err = cmd_err.results[ClockConsts.STDERR]
+                cmd_output = output if output else err
+                logging.info(f'Command Error!\nError message: "{cmd_output}"')
+            logging.info(f'Output type: {type(cmd_output)}')
+            logging.info(f'Output: {cmd_output}')
+
+        with allure.step('Convert output to string'):
+            logging.info('Convert output to string')
```

Comment:
Please remove this log. Avoid unnecessary logs and include only those which will help in analysis.

Reply:
I still see the log 'Convert output to string'. Can you please remove?